src/components/Server.js

```diff
@@ -174,7 +174,7 @@ class Server extends Router {
         any: {},
         get: {},
         post: {},
-        delete: {},
+        del: {},
```

Comment:
Probably not a good idea. `delete` is the standard.

Check [HTTP request methods](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods)

__DELETE__

    The DELETE method deletes the specified resource.

Reply:
This would be a problem if the change was to rename the Router.delete() -> Router.del() but @snowbldr simply fixed the underlying routes pool object to categorize delete routes with the "del" property which fixed the issue of route creation as uWebsockets.js uses TemplatedApp.del() instead of TemplatedApp.delete() for creating DELETE routes.

Reply:
Okay. Got it :smiley:

Reply:
I agree that del is weird, I wish uws.js had a delete function. I would add it but I'm too noob in c++, maybe it's a good opportunity to learn... Thanks for checking it.